src/app/cube/details/components/materials/components/notes/notes.component.html

```diff
@@ -1,3 +1,3 @@
 <section class='notes'>
-  <p tab-index='0' [innerHtml]="notes">{{ notes ? notes: 'This Learning Object does not have any notes' }}</p>
+  <p tab-index='0' [innerHtml]="notes">{{ notes }}</p>
```

Comment:
```suggestion
  <p tab-index='0' >{{ notes ? notes: 'This Learning Object does not have any notes' }}</p>
```
The issue should have been the `[innerHtml]` pipe and accessing notes through {{notes}}. The pipe overrides the {{}} logic so by removing the pipe, the bracket logic will work.